great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -5092,28 +5097,6 @@ def _determine_if_expectation_validation_result_include_rendered_content(
                 return False
         return include_rendered_content
 
-    @staticmethod
-    def _determine_save_changes_flag(save_changes: Optional[bool]) -> bool:
-        """
-        This method is meant to enable the gradual deprecation of the `save_changes` boolean
-        flag on various Datasource CRUD methods. Moving forward, we will always persist changes
-        made by these CRUD methods (a.k.a. the behavior created by save_changes=True).
-
-        As part of this effort, `save_changes` has been set to `None` as a default value
-        and will be automatically converted to `True` within this method. If a user passes in a boolean
-        value (thereby bypassing the default arg of `None`), a deprecation warning will be raised.
-        """
-        if save_changes is not None:
-            # deprecated-v0.15.32
```

Comment:
I think we need to update `test_deprecation.py` (it should have a threshold for the allowable number of deprecation warnings)

Reply:
Not quite clear to me how this test is supposed to work or what to change the threshold to. Currently, the threshold is set to 7, but I'm seeing 42 deprecations based on the policy that is being tested for.

I also don't think these would have been taken into account as part of that threshold as these warnings are not yet in violation of the policy